code/game/objects/structures/aliens.dm

```diff
@@ -317,6 +316,9 @@
 	spawn(rand(MIN_GROWTH_TIME, MAX_GROWTH_TIME))
 		Grow()
 
+/obj/structure/alien/egg/attack_alien(mob/living/carbon/alien/user)
+	return attack_hand(user)

```

Comment:
`. = attack_hand(user)`

Reply:
A hard return here is more consistent with other similar code.